src/main/java/uk/gov/hmcts/darts/common/repository/AnnotationRepository.java

```diff
@@ -53,4 +55,7 @@ public interface AnnotationRepository extends JpaRepository<AnnotationEntity, In
         """)
     List<AnnotationEntity> findByListOfHearingIdsAndUser(List<Integer> hearingIds, UserAccountEntity userAccount);
 
+    @Modifying
+    @Query("update AnnotationEntity ann set ann.deleted = true where ann.id = :id")
```

Comment:
probably want to update the last_modified_by also

Reply:
done